GliaWidgets/SecureConversations/Welcome/Theme+SecureConversationsWelcome.swift

```diff
@@ -274,7 +274,7 @@ extension Theme {
            return AttachmentSourceListStyle(
                items: [pickPhoto, takePhoto, browse],
                separatorColor: color.baseShade,
-               backgroundColor: Color.lightGrey
+               backgroundColor: Color.baseNeutral
```

Comment:
Do we need to use color from palette?
```suggestion
               backgroundColor: color.baseNeutral
```

Reply:
Why didn't we use it in the first place?

Reply:
fixed [f8b8b60](https://github.com/salemove/ios-sdk-widgets/pull/777/commits/f8b8b600ed2c3cccef497d0c35a59dee80a4cfea)